SPECS/patchelf/patchelf.spec

```diff
@@ -30,15 +30,15 @@ of an executable and change the RPATH of an executable or library.
 rm src/elf.h
 
 %build
-sh ./bootstrap.sh
+autoreconf -fi
```

Comment:
Why did this change? Can we not use the options set in bootstrap.sh https://github.com/NixOS/patchelf/blob/master/bootstrap.sh ?

Reply:
Keep using the bootstrap.sh from the source code.